app/assets/src/components/views/PipelineViz/pipeline_viz.scss

```diff
@@ -33,6 +33,11 @@
   word-break: break-word;
 }
 
+.notToggableStageButton {
```

Comment:
I think you can move this under .stageButton and change it to something like `&.disabled`